### PR TITLE
LPD-20544 Create Integration test for DefaultObjectEntryManagerImplTest related to ObjectEntryWithAccountEntryRestricted scope

### PR DESCRIPTION
**Pull request by jairmedeiros**
Related issue: https://liferay.atlassian.net/browse/LPD-20544

### CODE REVIEW

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java [hunk 1]
```diff
@@ -2202,6 +2228,32 @@ public void testGetObjectEntriesWithAccountEntryRestricted1()
 			ResourceConstants.SCOPE_COMPANY, String.valueOf(companyId),
 			role.getRoleId(), ActionKeys.VIEW);
 
+		// Regular roles permissions should bypass the account restriction
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java:2231]: Hey @jairmedeiros tests of this class are sorted alphabetically by comments, so to follow the pattern your use case should came before the use above,

Thank you!

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java [hunk 2]
```diff
@@ -1068,9 +1069,34 @@ public void testAddObjectEntry() throws Exception {
 	public void testAddObjectEntryWithAccountEntryRestricted1()
 		throws Exception {
 
-		// Account entry restricted scope
+		// Regular roles' company scope permissions should not be restricted by
+		// account entry
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java:1073]: @jairmedeiros sort this as well,


Thanks!

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]
@jairmedeiros see my comments,

nice job BTW! :tada:

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java (hunk 1 above)
Comment (jairmedeiros) [reply modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java]: Done!

Comment (jairmedeiros) [reply modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java]: Hi @MarinhoFeliphe, I removed this test, I noticed the test above is the same haha

**Comment by jairmedeiros** [comment]
@MarinhoFeliphe I sorted and remove a duplicated test

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

@jairmedeiros thanks!

**Comment by MarinhoFeliphe** [comment]
ci:forward

**Comment by MarinhoFeliphe** [comment]
https://github.com/brianchandotcom/liferay-portal/pull/148498#issuecomment-2018307395

**Comment by MarinhoFeliphe** [comment]
See: https://github.com/brianchandotcom/liferay-portal/pull/148498#issuecomment-2026198149

**Pull request closed without merging**